Remove requestId from subscriptionNotification and subscriptionNotificationError interface

wonsuk73 opened the pull request:
subscriptionNotification and subscriptionNotificationError might be
used after client request for ’subscribe’ action go success. It means
it's used under the one way notification from server to client. so I guess in this
situation requestId doesn’t have any meaning. How about removing this
from two interfaces?

wonsuk73 committed:
Remove requestId form subscriptionNotification and subscriptionNotificationError interface

subscriptionNotification and subscriptionNotificationError might be
used after client request for ’subscribe’ action go success. It means
it one way notification from server to client. so I guess in this
situation requestId doesn’t have any meaning. How about removing this
from two interfaces?

aShinjiroUrata commented:
Hi.
I agree on this. 
I was wondering the reason why to carry both requestId and subscriptionId.

wonsuk73 commented:
@aShinjiroUrata Thanks! for feedback! @acrofts84 @drkevg Could you review this? :-)

wonsuk73 committed:
Merge remote-tracking branch 'w3c/gh-pages' into gh-pages

wonsuk73 committed:
Change subscribe section

This PR propose to below changes for subscribe section
- Add timestamp to  subscribeSuccessResponse and subscribeErrorResponse
interface.
- Update an example in subscribe section to deliver clear meaning for
readers
